src/components/chat/chatBubble/ChatBubble.tsx

```diff
@@ -0,0 +1,30 @@
+import Image from '../../../assets/images/logo192.png';
+import '../../../assets/stylesheets/chat/chatBubble.scss';
+
+interface ChatBubbleProps {
+  isBotBubble: boolean;
+  message: string;
+  dateTime: string;
+}
+
+const ChatBubble = (props: ChatBubbleProps) => {
+  const { isBotBubble, message, dateTime } = props;
+
+  return (
+    <>
```

Comment:
Can you try removing this Fragment and just return the main div? It looks like it is redundant!

Reply:
Removed with fbee2c07f030c6214a7b046618b25eb8d61e66d8